server/src/test/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeAggregatorTests.java

```diff
@@ -3077,36 +3080,46 @@ public void testParentFactoryValidation() throws Exception {
                 addToDocument(0, document, createDocument("term-field", "a", "long", 100L));
                 indexWriter.addDocument(document);
             }
+            List<Releasable> releasables = new ArrayList<>();
```

Comment:
I think we don't put anything in the list other than `context`  - I think it's slightly more normal to declare `AggregationContext context = null` and then assign something to it and release the context, null or not.

Reply:
Or `try()` it, I guess.